docs/CONTRIBUTING.md

```diff
@@ -1,5 +1,52 @@
 # Contributing to Endless Sky
 
+## Build instructions
+
+**You can get a copy of the code either using "git clone," or using the repo's download button to obtain a [.ZIP archive](https://github.com/endless-sky/endless-sky/archive/master.zip).** The game's root directory, where your unzipped/`git clone`d files reside, will be your starting point for compiling the game.
+
+How you build Endless Sky will then depend on your operating system: [Windows](readme-windows.md], [Linux](readme-linux.md), [MacOS](readme-macos.md). Popular IDEs are supported through their CMake integration, for example: Visual Studio, Visual Studio Code, XCode, CodeBlocks, CLion, and many more.
+
+<details>
+<summary>Command line build instructions</summary>
+
+Here's a summary of every command you will need for development:
+
+```bash
+$ cmake --preset <preset>                     # configure project (only needs to be done once)
+$ cmake --build --preset <preset>-debug       # actually build Endless Sky (as well as any tests)
+$ ./build/<preset>/Debug/endless-sky          # run the game
```

Comment:
Isn't the executable's name different on every OS? I seem to recall it being an exe on Wind🤮ws.